### ISSUE TITLE
Does this method still work

### ISSUE THREAD

**Issue by Saya47** [wintersim/fiddler-spotify-adblock#1]
Hello I'd been using this method for a very long time now, but it doesn't seem to work anymore as I didn't change anything on my end. My speculation is that Spotify is now enforcing certificate checks and an unsecure certificate causes the application to not work anymore? Am I thinking right? Does this method still work for you? And wanted to thank you for sharing this method in the first place, it's been a good run.

**Comment by wintersim** [comment]
Yes it's still working for me on Fiddler Classic v5.0.20211.51073 and the latest Spotify update on Windows 10. I'm also using a Pi-Hole to block any other ads at DNS level. 
Your suspicion regarding the unsafe certificate checks might be valid. I intended to use mitmproxy (another https proxy), but I was never able to use Spotify with it and also assumed that they are noticing the unsafe cert.  

I don't think I can provide much assistance with your case, but maybe share your setup (versions, os, etc..) so we can figure something out.

**Comment by Saya47** [comment]
Hello sorry I could not respond sooner, I was so busy with university exams.
Anyhow, I finally figured out what was wrong just today!
I enabled Fiddler on system proxy settings, and opened a website on Google Chrome, it' funny I never paid attention to this, my certificates were invalidated! The website didn't load and showed this error:
NET::ERR_CERT_DATE_INVALID
So I reset my Fiddler certificates and reinstalled them and now it's working again!
I also noticed that all traffic rows on Fiddler were all gray instead of black.
Thanks a lot.
